package.json

```diff
@@ -44,7 +44,7 @@
 		"@storybook/addon-storyshots": "^5.0.11",
 		"@storybook/addon-viewport": "^5.0.11",
 		"@storybook/react": "^5.0.11",
-		"algoliasearch": "^3.32.1",
+		"algoliasearch": "^4.2.0",
```

Comment:
Have you been able to test this? I think I would caution against a major version bump of a dependency at this time, unless there's a specific reason for it.

Reply:
The reason for the bump was because I was getting really odd behavior using the old package. Algolia records with `isPublic: false` (as confirmed in the web dashboard) were coming down with `isPublic: true`. I wiped the Algolia indexes, re-uploaded records, tested in new browsers, and spent an hour or so debugging the strange behavior. It was a further problem because private information was leaking into results for non-logged-in users (the filters on isPublic were failing). 

Upgrading the package fixed the issue (though there was a migration guide that took some time to follow). I've tested the upload and search with the version bump and it seems to be good to go.